android/app/build.gradle

```diff
@@ -29,9 +30,19 @@ repositories {
     }
 
     google()
-    jcenter()
+    mavenCentral()
+    jcenter() {
+        content {
+            includeGroup("com.facebook.fbjni")
+            includeGroup("com.facebook.flipper")
+            includeGroup("com.facebook.fresco")
+            includeGroup("com.facebook.yoga")
+        }
+    }
```

Comment:
See issue #305.

Reply:
@arazabishov, @kelset, @sweggersen: I'm going to merge this if there are no objections.